dask_planner/Cargo.toml

```diff
@@ -9,19 +9,20 @@ edition = "2021"
 rust-version = "1.62"
 
 [dependencies]
-tokio = { version = "1.0", features = ["macros", "rt", "rt-multi-thread", "sync", "fs", "parking_lot"] }
-rand = "0.7"
-pyo3 = { version = "0.17.1", features = ["extension-module", "abi3", "abi3-py38"] }
 arrow = { version = "22.0.0", features = ["prettyprint"] }
-datafusion-sql = { git="https://github.com/apache/arrow-datafusion/", rev = "12.0.0-rc1" }
-datafusion-expr = { git="https://github.com/apache/arrow-datafusion/", rev = "12.0.0-rc1" }
-datafusion-common = { git="https://github.com/apache/arrow-datafusion/", rev = "12.0.0-rc1" }
-datafusion-optimizer = { git="https://github.com/apache/arrow-datafusion/", rev = "12.0.0-rc1" }
-uuid = { version = "0.8", features = ["v4"] }
+async-trait = "0.1.41"
```

Comment:
I ran `cargo tomlfmt` hence the diff here. I also added `log` and `env_logger` dependencies

Reply:
Doesn't need to be done in this PR, but would we want a check like this to be run as part of the pre-commit hooks?

Reply:
Yes, ideally we would add this to the pre commit hook and add checks in CI as well